### PR TITLE
Update documentation-style-guide.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Added guidelines for editing user guide as part of migration.

### CODE REVIEW

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 1]
```diff
@@ -305,3 +305,15 @@ The following example is a block of Go code.
 ## Using Markdown
 
 The majority of Mattermost technical documentation is written in `.rst`. However, there are some instances where Markdown is used, for example the Mattermost Handbook. You can read more about using Markdown in the [Formatting Text](https://docs.mattermost.com/help/messaging/formatting-text.html) section of the User's Guide.
+
+## Editing the Mattermost User Guide
+
+These are some additional tips for writing [Mattermost documentation for end users](https://docs.mattermost.com/guides/user.html).
+
+1.	Start fresh - begin by writing a draft in a Google document from scratch.
```
Comment (jasonblais) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: Is the added indentation on purpose?

**Review by jasonblais — APPROVED** [review]

**Review by jasonblais — CHANGES_REQUESTED** [review]
Meant to comment instead of approve, see above

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 1 above)
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]:
```suggestion
1. Start fresh - begin by writing a draft in a Google document from scratch.
```

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md [hunk 2]
```diff
@@ -305,3 +305,15 @@ The following example is a block of Go code.
...
+2. Start the document by describing the benefits of the feature for end users ([see this example](https://docs.mattermost.com/help/messaging/attaching-files.html)).
+3.	Compare to the original document to add any important information or tips that you forgot to include in your draft.
```
Comment (justinegeffen) [inline operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]:
```suggestion
3. Compare to the original document to add any important information or tips that you forgot to include in your draft.
```

File: operations/operations/publishing/publishing-guidelines/documentation-style-guide.md (hunk 1 above)
Comment (justinegeffen) [reply operations/operations/publishing/publishing-guidelines/documentation-style-guide.md]: Nope, definitely not. Thanks for catching it - it only showed up when I was in "suggestion" mode for some reason.

**Review by jasonblais — APPROVED** [review]

**Pull request merged**